test/benchmark_test.cc

```diff
@@ -176,7 +176,7 @@ static void BM_ParallelMemset(benchmark::State& state) {
     delete test_vector;
   }
 }
-BENCHMARK(BM_ParallelMemset)->Arg(10 << 20)->ThreadRange(1, 4);
+BENCHMARK(BM_ParallelMemset)->Explicit({10 << 20})->ThreadRange(1, 4);
```

Comment:
No, please add standalone test.
Also, you may want to mention it in docs next to `Arg()`